src/widgets/radio.jsx

```diff
@@ -441,7 +440,12 @@ var RadioEditor = React.createClass({
                     <PropCheckBox label="Auto-none of the above"
                                   labelAlignment="right"
                                   noneOfTheAbove={this.props.noneOfTheAbove}
-                                  onChange={this.props.onChange} />
+                                  onChange={(e) => {
+                                    var rand = seededRNG(this.props.problemNum)();
+                                    var index = Math.floor(rand * this.props.choices.length);
```

Comment:
I think we should get rid of the auto none of the above altogether, ideally. I think it will make both the user interface and the code a bit easier to reason about :).